app/src/main/java/com/zulip/android/activities/LoginActivity.java

```diff
@@ -233,15 +233,17 @@ public void onClick(View v) {
             case R.id.google_sign_in_button:
                 connectionProgressDialog.show();
                 saveServerURL();
+                Toast.makeText(this, getString(R.string.logging_into_server, ZulipApp.get().getServerURI()), Toast.LENGTH_SHORT).show();
```

Comment:
It was there since before so I added that line as it is!
Should I remove that, it seems correct though as we are logging in to a server and we should give them a message that goes like (Logging into server :{server url}) ?